Server/User/routehelper.py

```diff
@@ -111,10 +124,10 @@ def createUser(data):
 				error = "That email is already in use."
 			else:
 				error = str(e)
+		else:
+			error = str(e)
```

Comment:
This looks weird.. so maybe just ignore, but what about putting error = str(e) at the top and removing the else's?  If an if is caught, overwrite error